opentracing-noop/src/main/java/io/opentracing/noop/NoopTracer.java

```diff
@@ -25,6 +26,16 @@
 final class NoopTracerImpl implements NoopTracer {
     final static NoopTracer INSTANCE = new NoopTracerImpl();
 
+    private final ActiveSpanSource spanSource;
+
+    public NoopTracerImpl() {
```

Comment:
It might be better to keep it as package level visibility. The same for the second constructor.